linux/user-and-file-management/bash-environment/using-the-terminal.md

```diff
@@ -0,0 +1,69 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+  - terminal
+  - bash
+  - shell
+
+---
+
+# Using the Terminal
+
+---
+## Content
+
+Before we go on, let's open the terminal. There are a few ways of doing this:
+- pressing `CTRL + ALT + T`
+- right-click anywhere in a directory/desktop and press `Open terminal`
+- press the `Super` key (AKA the `Windows` key) and search for `terminal`
+
+Now that it's open, go ahead and type:
+
+```bash
+echo "Enki is cool!"
+Enki is cool!
+```
```

Comment:
This kind of implies that the user should type in the `echo` command and the text below. How about we split them?

````suggestion
Now that it's open, go ahead and type:

```bash
echo "Enki is cool!"
```

You should see the text display in the terminal:

```
Enki is cool! 
```
````